x-pack/packages/kbn-ecs-data-quality-dashboard/impl/data_quality/use_results_rollup/helpers.ts

```diff
@@ -31,7 +35,21 @@ export const getTotalDocsCount = (
 
   // only return the total when all `PatternRollup`s have a `docsCount`:
   return allRollupsHaveDocsCount
-    ? allRollups.reduce((acc, { docsCount }) => acc + (docsCount ?? 0), 0)
+    ? allRollups.reduce((acc, { docsCount }) => acc + Number(docsCount), 0)
```

Comment:
Is there ever a scenario where we would want to show partial results if one of the indices didn't return a value? Not necessary now, but just a thought. We'll see how often users run into this scenario...